src/jupyterhub_config/custom_spawner.py

```diff
@@ -14,4 +16,43 @@ def start(self):
         variables, and sets the notebook directory before starting the server.
         """
 
-        return super().start()
\ No newline at end of file
+        username = self.user.name
```

Comment:
I assume this is coming from the superclass

Reply:
Yea

Reply:
:+1: